modules/data/pending_post.py

```diff
@@ -140,49 +140,68 @@ def get_votes(self, vote: bool) -> int:
                                     where="g_message_id = %s and group_id = %s and is_upvote = %s",
                                     where_args=(self.g_message_id, self.group_id, vote))
 
-    def get_list_admin_votes(self, vote: bool) -> Optional[list[str]]:
+    def get_list_admin_votes(self, vote: 'bool | None' = None) -> 'Optional[list[str] | bool]':
```

Comment:
I guess this style of type-hinting as a string is used to be python 3.9 compatible. I guess it's fine, although this means that some part of the application use `Optional` and `Union` while others will use `|`

Reply:
Also, it should probably be

```diff
- Optional[list[str] | bool]
+ Optional[list[str | bool]]
```